Makefile.am

```diff
@@ -222,6 +222,11 @@ endif # LZ4_ENABLED
 
 endif # UV_ENABLED
 
+if BACKTRACE_ENABLED
+libraft_la_CFLAGS += -DRAFT_ASSERT_WITH_BACKTRACE
+libraft_la_LDFLAGS += -lbacktrace
```

Comment:
Compilation will fail if `libbacktrace` is not found, probably a bit cleaner to detect it during the configure step itself, the `libuv` in `configure.ac` is probably the clearest.

Reply:
Fair enough, I can live with a failure further down the road, it's a debugging feature anyway.